lib/manageiq/messaging/stomp/common.rb

```diff
@@ -1,27 +1,19 @@
 module ManageIQ
   module Messaging
-    module Common
-      def self.included(base)
-        base.extend ClassMethods
-      end
-
-      module ClassMethods
+    module Stomp
+      module Common
         private
 
-        def logger
-          ManageIQ::Messaging.logger
-        end
-
-        def raw_publish(client, address, body, headers)
-          client.publish(address, encode_body(headers, body), headers)
-          logger.info("Address(#{address}), msg(#{body.inspect}), headers(#{headers.inspect})")
+        def raw_publish(address, body, headers)
+          publish(address, encode_body(headers, body), headers)
+          logger.info("Published to address(#{address}), msg(#{body.inspect}), headers(#{headers.inspect})")
```

Comment:
What if the body is really big?  Should it be truncated?  What if the body has sensitive information?